project-base/storefront/components/Blocks/Product/ProductsSlider.tsx

```diff
@@ -7,51 +8,58 @@ import useTranslation from 'next-translate/useTranslation';
 import { RefObject, createRef, useEffect, useRef, useState } from 'react';
 import { useSwipeable } from 'react-swipeable';
 import { twJoin } from 'tailwind-merge';
+import { twMergeCustom } from 'utils/twMerge';
 import { isWholeElementVisible } from 'utils/ui/isWholeElementVisible';
 import { wait } from 'utils/wait';
 
 export type ProductsSliderProps = {
     products: TypeListedProductFragment[];
     gtmProductListName: GtmProductListNameType;
     gtmMessageOrigin?: GtmMessageOriginType;
-    isWithSimpleCards?: boolean;
+    isWithArrows?: boolean;
+    wrapperClassName?: string;
+    productItemProps?: Partial<ProductItemProps>;
 };
 
-const productTwClass = 'snap-center border-b-0 md:snap-start';
+const productTwClass = 'snap-center border-b-0 md:snap-start mx-1.5 first:ml-0 last:mr-0';
 
 export const ProductsSlider: FC<ProductsSliderProps> = ({
     products,
     gtmProductListName,
     gtmMessageOrigin = GtmMessageOriginType.other,
     tid,
-    isWithSimpleCards,
+    wrapperClassName,
+    isWithArrows = true,
+    productItemProps,
 }) => {
     const { t } = useTranslation();
-    const maxVisibleSlides = isWithSimpleCards ? 3 : 4;
+    const maxVisibleSlides = 4;
```

Comment:
Why was this number of slides changed?

Reply:
Change in the size of product items, causes change of this number
We still need to refactor the product slider as it is hard to use in other places on the eshop, so it will be refactored in the future

Reply:
Well, this change (together with other changes in this PR) breaks the layout of some product slider items (e.g. in add to cart popup). I think it should be fixed.

<img width="764" alt="image" src="https://github.com/shopsys/shopsys/assets/55181354/330204d3-4c75-4058-aebc-c75ca5ed7775">